### ISSUE TITLE
Centered vs Forward marginal effects

### ISSUE THREAD

**Issue by vincentarelbundock** [vincentarelbundock/marginaleffects#945]
Currently, the default is centered. Do we want to change to forward by default?

1. More intuitive: "The effect of an increase from observed value"
2. Cheaper: we already have the basic prediction, so we only need to compute one more.

**Comment by lang-benjamin** [comment]
Just my 2 cents... This "centered" version makes sense if the goal is to approximate the derivative. The value of the function at x is however not used in this approach - and to me this is a problem if  we are not interested in the slope but in a comparison with some incremental value not near zero. As you said, it is not intuitive then.

The fact that it may be quite different than centered in non-linear functions of course something to keep in mind, but I would not call it "Con" as it is just a definition (your wording implicitly suggests that the centered version is the preferred approach). 
(to be fair: On the other hand, it would be nice if comparisons() with an increment of about 1 resembles the slopes() result, which may not be the case if slopes() still uses the two-sided approximation formula. Still, I think the forward version  makes more sense for the comparisons() function).

**Comment by vincentarelbundock** [comment]
Yeah, I think I agree with all this. Thanks for chiming in because it reinforces where I was leaning.

I could also implement a global option to allow users to.change easily between those.

**Comment by vincentarelbundock** [comment]
Done: https://github.com/vincentarelbundock/marginaleffects/commit/981f5f068812b82f84c0906744215b92b484a29b
